spec/helpers-spec.coffee

```diff
@@ -1,38 +1,46 @@
-Helpers = require '../lib/helpers'
-
-describe "The Results Validation Helper", ->
-  it "should throw an exception when nothing is passed.", ->
-    expect( -> Helpers.validateMessages()).toThrow()
-  it "should throw an exception when a String is passed.", ->
-    expect( -> Helpers.validateMessages('String')).toThrow()
-  it "should throw an exception when a result's type is missing.", ->
-    results = [{}]
-    expect( -> Helpers.validateMessages(results)).toThrow()
-  it "should return the results when validated.", ->
-    results = [{type: 'Type'}]
-    expect(Helpers.validateMessages(results)).toBeUndefined()
-
-describe "The Linter Validation Helper", ->
-  it "should throw an exception when grammarScopes is not an Array.", ->
-    linter = {
-      grammarScopes: 'not an array'
+describe 'helpers', ->
+  helpers = require('../lib/helpers')
+  beforeEach ->
+    atom.notifications.clear()
+  describe '::error', ->
+    it 'adds an error notification', ->
+      helpers.error(new Error())
+      expect(atom.notifications.getNotifications().length).toBe(1)
+  describe '::messageKey', ->
```

Comment:
This method needs more tests. You didn't test `message.html`, `message.range`, `message.filePath` or `message.type` at all. You didn't test that it formatted the message as you expect.

The point of tests is to ensure that you or others don't accidentally break things. I could change the definition of `helpers.messageKey` to return the string `foo` and these tests would pass.